src/ModelLock.php

```diff
@@ -25,6 +25,8 @@ class ModelLock extends Model
      */
     protected static function boot()
     {
+        parent::boot();
```

Comment:
We should consider moving this 5.7 into into a PR of its own.

Reply:
Thank You 👍